models/incremental/fct_dbt__model_executions.sql

```diff
@@ -35,10 +35,7 @@ model_executions_with_materialization as (
         models.name
     from model_executions_incremental
     left join models on
-        (
-            model_executions_incremental.command_invocation_id = models.command_invocation_id
-            or model_executions_incremental.dbt_cloud_run_id = models.dbt_cloud_run_id
-        )
+        model_executions_incremental.command_invocation_id = models.command_invocation_id
```

Comment:
I just re-read the associated issue. Does removing the join on `dbt_cloud_run_id` not just undo https://github.com/brooklyn-data/dbt_artifacts/pull/38 and make this incompatible with dbt Cloud? Running the upload macro causes the manifest to be re-created and gives it a new command_invocation_id per https://github.com/brooklyn-data/dbt_artifacts/issues/53.

Reply:
Depends on our intent.
1. Based on my reading of #19 and #38- the idea was to only upload the manifest once - then allow ever set of related results to upload. That means `artifact_run_id` makes sense as the key for joining here because you can rely on there only being one.
2. The intent here changes that and assumes each run of the upload macro creates a new manifest - which is then directly joined to. That means that `command_invocation_id` is the right key to join on here.

To avoid a cartesian join I think we need to be clear on what we expect to be duplicated and what we expect to be unique in the dbt cloud "multi-upload" case.

The more I think about this - re-uploading the manifest does seem a bit wasteful and maybe rather than this PR - I need to modify the V2 upload so that the manifest gets overwritten rather than duplicated. @NiallRees thoughts?

Reply:
If we can achieve a state where we only have one of each unique manifest, that is definitely advantageous. Did you see  my note
> Running the upload macro causes the manifest to be re-created and gives it a new command_invocation_id per https://github.com/brooklyn-data/dbt_artifacts/issues/53.

Reply:
Ok - let me rethink this PR. Will change the intent.

Reply:
Hmmm - actually this could be tricky. Will slack you.